### DISCUSSION TITLE
ERRCODE: SC_ERR_AHO_CORASICK(174)] - Just ran out of space in the queue

### DISCUSSION THREAD

**Discussion by cac0ns3c** [Security-Onion-Solutions/securityonion#5738] [Unsupported Versions]
Hi Everyone,

so-suricata is crashing every 15 minutes.
Tried to debug it but didn't find any solution, any thoughts about it ?

Thanks.

**Comment by clairmont32** [comment]
on the sensor, whats the result of `docker exec so-suricata /opt/suricata/bin/suricata -T`. what is in /opt/so/log/suricata/suricata.log?
